core/core-services-impl/src/test/java/com/sflpro/notifier/services/notification/impl/email/EmailNotificationProcessorImplTest.java

```diff
@@ -129,6 +160,7 @@ public void testProcessNotificationForWhenProviderTypeIsMandrillAndFail() {
         notification.setId(notificationId);
         notification.setState(NotificationState.CREATED);
         notification.setProviderType(NotificationProviderType.MANDRILL);
+        notification.setTemplateName("YoTemplate");
```

Comment:
Is the template name mandatory here?

Reply:
Yes. I've removed it from helper. The sender method determines if the Email model is templated or simple by this field. So for simple model sending test we are not providing this field and in for the one with templated model we do